### PR TITLE
Add to the CIS app readme information about dependency on monitoring app to enable alerts, update image tags

### PR DESCRIPTION
**Pull request by prachidamle**
Adding information to the CIS v2 app readme about how to enable alerts. Please review the wording.

Also updating image tags to deliver: https://github.com/rancher/cis-operator/issues/69

### CODE REVIEW

**Review by mantis-toboggan-md — DISMISSED** [review]

File: packages/rancher-cis-benchmark/charts/app-readme.md [hunk 1]
```diff
@@ -10,3 +10,6 @@ This chart installs the following components:
 - Scans - A scan is a CRD (`ClusterScan`) that defines when to trigger CIS scans on the cluster based on the defined profile. A report is created after the scan is completed.
 - Profiles - A profile is a CRD (`ClusterScanProfile`) that defines the configuration for the CIS scan, which is the benchmark versions to use and any specific tests to skip in that benchmark. This chart installs a few default `ClusterScanProfile` custom resources with no skipped tests, which can immediately be used to launch CIS scans.
 - Benchmark Versions - A benchmark version is a CRD (`ClusterScanBenchmark`) that defines the CIS benchmark version to run using kube-bench as well as the valid configuration parameters for that benchmark. This chart installs a few default `ClusterScanBenchmark` custom resources.
+- Alerting Resources - Rancher CIS Benchmarks application lets you run a cluster scan on a schedule and also send alerts when scheduled scans finish runs.
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/app-readme.md:13]:
```suggestion
- Alerting Resources - Rancher's CIS Benchmark application lets you run a cluster scan on a schedule, and send alerts when scans finish.
```

File: packages/rancher-cis-benchmark/charts/app-readme.md [hunk 2]
```diff
@@ -10,3 +10,6 @@ This chart installs the following components:
...
+    - If you want to enable alerts to be delivered when a cluster scan completes, you need to ensure that [Rancher's Monitoring and Alerting](https://rancher.com/docs/rancher/v2.x/en/monitoring-alerting/v2.5/) application is pre-installed and the [Receivers and Routes](https://rancher.com/docs/rancher/v2.x/en/monitoring-alerting/v2.5/configuration/#alertmanager-config) are configured to send out alerts.
+    - Additionally, you need to set `alerts: true` in Values YAML while installing/upgrading this Rancher CIS Benchmarks chart.
```
Comment (cbron) [inline packages/rancher-cis-benchmark/charts/app-readme.md:15]:
```suggestion
    - Additionally, you need to set `alerts: true` in the Values YAML while installing or upgrading this chart.
```

**Review by cbron — DISMISSED** [review]
Just a nit, otherwise lgtm. Denise may want to review separately.

**Review by cbron — APPROVED** [review]

**Comment by prachidamle** [comment]
Merging this, it has been approved by @deniseschannon as well offline

**Pull request merged**